genai_stack/genai_platform/settings/config.py

```diff
@@ -0,0 +1,17 @@
+import os
+from configparser import ConfigParser
+
+# 
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Parse configuration
+GENAI_STACK_CONF = os.path.join(BASE_DIR,"genai_stack.conf")
```

Comment:
We should make this config file dynamically variable based on the user's working directory etc.